src/mixins/animation-frame.html

```diff
@@ -0,0 +1,64 @@
+<script>
+  window.requestAnimFrame = (function(){
+    return  window.requestAnimationFrame ||
+      window.webkitRequestAnimationFrame ||
+      window.mozRequestAnimationFrame    ||
+      function( callback ){
+        window.setTimeout(callback, 1000 / 60);
+      };
+  })();
+
+  const AnimationFrame = subclass => class extends subclass {
```

Comment:
Initially, I wanted to have one mixin with all functions requesting animation frame